addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -84,9 +84,10 @@ class HrApplicant(models.Model):
                                     help="Stage of the applicant before being in the current stage. Used for lost cases analysis.")
     categ_ids = fields.Many2many('hr.applicant.category', string="Tags")
     company_id = fields.Many2one('res.company', "Company", compute='_compute_company', store=True, readonly=False, tracking=True)
-    user_id = fields.Many2one(
-        'res.users', "Recruiter", compute='_compute_user', domain="[('share', '=', False), ('company_ids', 'in', company_id)]",
+    recruiter = fields.Many2one(
```

Comment:
Same here, should be `recruiter_id`.
```suggestion
    recruiter_id = fields.Many2one(
```